minecraft/minecraft.json

```diff
@@ -189,6 +219,40 @@
         "java -jar fabric-installer.jar server -mcversion ${version} -downloadMinecraft -noprofile"
       ]
     },
+    {
+      "if": "modlauncher == 'neoforge'",
+      "type": "neoforgedl",
+      "target": "installer.jar",
+      "version": "${neoforgebuild}",
+      "minecraftVersion": "${version}",
+      "outputVariable": "resolvedNeoForgeVersion"
+    },
+    {
+      "if": "modlauncher == 'neoforge' && env == 'host'",
+      "type": "command",
+      "commands": [
+        "java${javaversion} -jar installer.jar --installServer"
+      ]
+    },
+    {
+      "if": "modlauncher == 'neoforge' && env != 'host'",
+      "type": "command",
+      "commands": [
+        "java -jar installer.jar --installServer"
+      ]
+    },
+    {
+      "if": "modlauncher == 'neoforge' && file_exists('neoforge-' + neoforgebuild + '-shim.jar')",
```

Comment:
This doesn't exist as-is with NeoForge, and they seem to be against adding it.

Reply:
Right! I'll remove that, I do know they have a -beta as well if we want to add that or not